src/schnorr.rs

```diff
@@ -88,9 +91,17 @@ impl Signature {
     }
 
     /// Returns a signature as a byte array.
-    #[inline]
+    #[deprecated(since = "0.30.0", note = "Use `to_byte_array` instead.")]
```

Comment:
We usually use `TBD` in since and mass-replace it during release but there isn't anything wrong with this.